R/webApiDataSources.R

```diff
@@ -0,0 +1,118 @@
+# @file dataSources
+#
+# Copyright 2019 Observational Health Data Sciences and Informatics
+#
+# This file is part of StudyManagement
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#' Get the results from individual Achilles reports by data Source definition expression
+#'
+#' @details
+#' Obtain the JSON expression from WebAPI for a given dataSource
+#'
+#' @param baseUrl        The base URL for the WebApi instance, for example:
+#'                       "http://server.org:80/WebAPI".
+#' @param sourceKey      The source key for a CDM instance in WebAPI, as defined in the
+#'                       Configuration page
+#' @param reportType     The precomputed report type in Atlas (Achilles report e.g. dashboard, datadensity, person etc)
+#' @return
+#' A list of object with pre computed Achilles results for the sourceKey and Achilles report
+#'
+#' @examples
+#' \dontrun{
+#' # This will obtain pre-computed summary regarding dataSource
+#'
+#' getDataSourceReportForSourceKey(baseUrl = "http://server.org:80/WebAPI", dataSource = 'dataSource', reportType = 'dashboard')
+#' }
+#'
+#' @export
+
+getDataSourceReportForSourceKey <- function(baseUrl,
```

Comment:
What is the source key, and why is it important enough to mention in the function name?